packages/main/src/themes/base/SliderBase-parameters.css

```diff
@@ -9,14 +9,14 @@
 	--_ui5_slider_handle_width: 1.625rem;
 	--_ui5_slider_handle_border: solid 0.125rem var(--sapField_BorderColor);
 	--_ui5_slider_handle_background: var(--sapButton_Background);
-	--_ui5_range_slider_handle_background: rgba(var(--sapButton_Background), 0.25);
+	--_ui5_range_slider_handle_background: rgba(255, 255, 255, 0.25);

```

Comment:
Leave a comment how it should have been if we don't support IE, so we can bring it back afterwards.
Because, now this becomes untameable, as does not depend on a variable, but on a hardcoded value.

Reply:
The issue is that the parsed from UI5 Fiori variable is using hex color value - #fff and IE doesn support color with alpha channel property with hex value (e.g - rgba(#fff, 0.25) but only rgb color value (rgba(255, 255, 255, 0.25), if we don't support IE we can restore the variable, or if there is another way to use it in IE with an alpha channel opacity.

While in UI5 the styling of the handles are the same, the problem doesn't exist because the LESS preprocessor correctly converts the hex value of the color to rgb values.

The 'opacity' CSS property has a different behaviour and it's isn't suitable in our case - it affects borders too.

Reply:
Maybe I was not clear, I understand the issue, just asked to leave a comment:

Something like:
--_ui5_range_slider_handle_background: rgba(255, 255, 255, 0.25);
/* --_ui5_range_slider_handle_background: rgba(var(--sapButton_Background), 0.25); - does not work on IE */

to help us bring it back once we drop the IE support.